classes/game.rb

```diff
@@ -0,0 +1,30 @@
+class Game
```

Comment:
I think you missed looking at the diagram, that's why you did not create the inheritance relationship between `Game` and `Item` classes, please Add the relationship by adding something like this: 
`class Game < Item`